### DISCUSSION TITLE
Frigate LXC

### DISCUSSION THREAD

**Discussion by sqorpo** [tteck/Proxmox#3730] [Help Needed] [answered]
Starting this as a new issue due to the other thread being "answered". My Frigate lxc crashed after trying to get hardware acceleration working on my setup. I thought I corrupted the config, but after seeing other people having the same issue, I'm suspecting it was just the reboot that did it. I've tried rerunning the script on 2 different pve installs on 2 different machines with the same result. Tried all default options, advanced, and even a complete fresh pve install with only the Frigate script. Still same issue.. Frigate will not load yml. Not the example that it comes with or any new ones created. Go2rtc is unreachable. Not sure if it's even running. Probably other issues too, but without Frigate loading yml that's kind of a full stop

**Comment by tteck** [comment]
I don't use Frigate, but I just installed it to check for issues.

```
    ______     _             __
   / ____/____(_)___ _____ _/ /____
  / /_  / ___/ / __ `/ __ `/ __/ _ \
 / __/ / /  / / /_/ / /_/ / /_/  __/
/_/   /_/  /_/\__, /\__,_/\__/\___/
             /____/

Using Advanced Settings
Using Distribution: debian
Using debian Version: 11
Using Container Type: 0
Using Root Password: Automatic Login
Container ID: 111
Using Hostname: frigate
Using Disk Size: 20
Allocated Cores: 4
Allocated RAM: 4096
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Frigate LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.114.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Python3 Dependencies
 ✓ Installed Node.js
 ✓ Installed go2rtc
 ✓ Set Up Hardware Acceleration
 ✓ Stop spinner to prevent segmentation fault
 ✓ Installed Frigate v0.14.1
 ✓ AVX Support Detected
 ✓ Installed Openvino Object Detection Model
 ✓ Installed Coral Object Detection Model
 ✓ Built Nginx
 ✓ Installed Tempio
 ✓ Configured Services
 ✓ Customized Container
 ✓ Cleaned
 ✓ Set Container to Normal Resources
 ✓ Completed Successfully!

Frigate should be reachable by going to the following URL.
         http://192.168.86.250:5000 

go2rtc should be reachable by going to the following URL.
         http://192.168.86.250:1984 
```

The installation process seems to complete successfully, but nothing is working, not even logs.

![Screenshot 2024-09-15 10 33 18 AM](https://github.com/user-attachments/assets/f35f50a2-16fe-466c-b14b-53c908ca54fd)

@remz1337 with the amount of issues coming in, I'm thinking about discontinuing this script. Not worth the headache.

**Reply by floxigen** [reply]
same for logs / monitor 
![Capture d’écran du 2024-09-15 17-06-22](https://github.com/user-attachments/assets/399e944e-ca81-4c94-b63a-2f5f1a253855)
![Capture d’écran du 2024-09-15 17-06-48](https://github.com/user-attachments/assets/43585af5-37d7-4b9f-8523-e8659342c1f5)
![Capture d’écran du 2024-09-15 17-07-02](https://github.com/user-attachments/assets/58502e30-8438-4650-83b6-d4163bbcf1d3)

**Reply by remz1337** [reply]
@tteck it is indeed quite a complex system to setup and maintain, so I understand your decision to remove it. Although, it would be nice if you could redirect user to my repo for Frigate script/support.

@sqorpo  @floxigen feel free to open an issue in my repo https://github.com/remz1337/Proxmox

**Reply by tteck** [reply, marked as answer]
Fixed. The issue was that the script is installing Frigate via the release package, but the script was fetching the dev version of the `install_deps.sh` file. This was asking for issues. I'm going to add it back to the website.

**Reply by sqorpo** [reply]
> Fixed. The issue was that the script is installing Frigate via the release package, but the script was fetching the dev version of the `install_deps.sh` file. This was asking for issues. I'm going to add it back to the website.

Thank you so much.. So glad you were able to get this fixed. Your scripts are very helpful and much appreciated...

**Reply by wildchai** [reply]
> @tteck it is indeed quite a complex system to setup and maintain, so I understand your decision to remove it. Although, it would be nice if you could redirect user to my repo for Frigate script/support.
> 
> @sqorpo @floxigen feel free to open an issue in my repo https://github.com/remz1337/Proxmox

@remz1337 I see in your repo there is an update option, can this be used for original tteck's Frigate script? and how is your method differs from tteck's? appreciate the help.

**Reply by remz1337** [reply]
@wildchai yes the update path should work with the default installation from tteck. The only differences regarding frigate are the update path and support for TensorRT (object detection using GPU passthrough)
